<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seev::HomePageWidget</class>
 <widget class="QWidget" name="seev::HomePageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="newSearchBox">
     <property name="title">
      <string>New Search</string>
     </property>
     <layout class="QGridLayout" name="newSearchLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="searchEditDlgBut">
        <property name="whatsThis">
         <string>If the above editing space is too small, click this for a dedicated editor.</string>
        </property>
        <property name="text">
         <string>Edit Search in a Separate Dialog...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="initSearchBut">
        <property name="text">
         <string>Start Search</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QScrollArea" name="predScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="seev::OriePredSelector" name="predWidg"/>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="openStartPathDlgBut">
        <property name="whatsThis">
         <string>By default search for all indexed file. You can choose a subset of it.</string>
        </property>
        <property name="text">
         <string>Edit Start Paths</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="advOptBox">
        <property name="title">
         <string>Advanced Options</string>
        </property>
        <layout class="QFormLayout" name="_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Index Update Interval</string>
           </property>
           <property name="buddy">
            <cstring>updIntSpin</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="updIntSpin">
           <property name="whatsThis">
            <string>Short interval may burden your system, and long intervals may have trouble keeping the index up-to-date.</string>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="singleStep">
            <number>300</number>
           </property>
           <property name="value">
            <number>3600</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="openConfWidgBut">
           <property name="text">
            <string>Indexing Settings</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="typeOrieCmdBut">
           <property name="text">
            <string>Type `orient` Command (Expert)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="orieCmdEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Monospace</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>-print, -delete and -exec nodes are prohibited in SearchEverywhere as they may lead to unexpected actions.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="seevConfPathBut">
           <property name="whatsThis">
            <string>The default configuration path works for the most part.</string>
           </property>
           <property name="text">
            <string>Select seev Conf Path</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="seevConfPathEdit">
           <property name="whatsThis">
            <string>The default configuration path works for the most part.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="updateDbBut">
        <property name="whatsThis">
         <string>Usually the update on startup and auto updates are sufficient. Only update if you want the searches to manifest your recent changes to filesystem.</string>
        </property>
        <property name="text">
         <string>Scan Filesystem</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="showAdvChk">
        <property name="text">
         <string>Show Advanced</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="__z2">
     <property name="text">
      <string>Saved Searches - Double Click to Load</string>
     </property>
     <property name="buddy">
      <cstring>savedSearchLst</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="savedSearchLst"/>
   </item>
   <item>
    <widget class="QPushButton" name="delSavedSearchBut">
     <property name="text">
      <string>Delete Selected</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>seev::OriePredSelector</class>
   <extends>QWidget</extends>
   <header>widgets/oriepredselector.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>showAdvChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>advOptBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
